docs/tutorials/multimodal/object_detection/inference/detection_inference_quick_start.md

```diff
@@ -0,0 +1,177 @@
+# AutoMM Detection - Inference with Pretrained Faster R-CNN on VOC Format Dataset
+:label:`sec_automm_detection_infer_coco`
+
+In this section, we show an quick-start example to run inference on a small dataset (Tiny Motorbick) that is in COCO Format.
+
+## Prepare data
+```python
+import os
+import time
+
+from autogluon.core.utils.loaders import load_zip
+```
+
+The data file stored in the cloud is located at:
+```python
+zip_file = "s3://automl-mm-bench/object_detection_dataset/tiny_motorbike_coco.zip"
+```
+
+Now let's download the dataset
+```python
+download_dir = "./tiny_motorbike_coco"  # specify a target download dir to store this dataset
+
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+data_dir = os.path.join(download_dir, "tiny_motorbike")
+train_path = os.path.join(data_dir, "Annotations", "coco_trainval.json")
```

Comment:
why coco_trainval and coco_test for tiny_motorbike?

Reply:
I think it was named coco for its format. @FANGAreNotGnu do you have any insights into this?